### DISCUSSION TITLE
Define and export often used constants.

### DISCUSSION THREAD

**Discussion by StanleyDing** [wevm/viem#447] [Idea / Feature Request]
In ethers.js, there are [constants](https://docs.ethers.org/v5/api/utils/constants/) for user to import, can viem do the same? I think it's relative easy to implement.

**Comment by Melancholism** [comment]
Has it been implemented yet? I think it is necessary.
Appreciate your efforts.

**Comment by xtools-at** [comment]
i've added all missing ethers-v5 and -v6 constants in this PR: https://github.com/wagmi-dev/viem/pull/1475
